cli/src/wad/ctx.rs

```diff
@@ -82,9 +82,16 @@ impl<'a> WadContext<'a> {
         let mut inflater = Inflater::new();
 
         for (idx, (path, file)) in self.journal.iter().enumerate() {
+
             // Extract the file range we care about.
             let contents = Self::file_contents(&self.mapping, file);
 
+            // TODO: fix this for repacking
+            // this skips over unpached files which are just 0s * size
+            if contents[0..4] == [0_u8; 4] {
```

Comment:
this shouldn't be possible for any valid files since zlib header is probably over 4 bytes and for non-compress files their file magic is probably over 4 bytes

we can probably still do it just in case though